simulator-core/src/main/java/com/evbox/everon/ocpp/simulator/station/Station.java

```diff
@@ -31,17 +31,16 @@
 @Slf4j
 public class Station {
 
-    private final OkHttpClient DEFAULT_HTTP_CLIENT;
+    private final OkHttpClient defaultHttpClient;
 
     private final SimulatorConfiguration.StationConfiguration configuration;
 
     private final StationStore state;
-    private volatile StationStoreView stationStoreView;
+    private StationStoreView stationStoreView;
```

Comment:
I'd rather keep it as is, cause stateView will likely be read and written by different threads. Sonar complaint is not relevant in this case. Consider putting ignore to this field to prevent sonar warnings or simply leave as it was

Reply:
Done